src/types/api.ts

```diff
@@ -343,6 +343,12 @@ export type APIAnnotationData = {
    * This prop will be present only if `at_mentions` is enabled.
    */
   mentions?: Mention[];
+
+  /**
+   * The list of actions current user can perform over this annotation.
+   * Possible values are: `moderate`.
```

Comment:
Perhaps make the known actions part of the type, eg. `'moderate' | string`? This won't change type inference but it will act as documentation.